### PR TITLE
Remove hash from url

### PR DESCRIPTION
**Pull request by larajanecka**
Removes query hash

### CODE REVIEW

File: src/js/client.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import { Router, Route, IndexRoute, hashHistory, IndexLink } from "react-router";
+import { Router, useRouterHistory } from 'react-router'
+import { Route, IndexRoute, IndexLink } from "react-router";
```
Comment (StevenLocke) [inline src/js/client.js]: Any reason we're importing from both `'react-router'` and `"react-router"`?

File: src/js/client.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import { Router, Route, IndexRoute, hashHistory, IndexLink } from "react-router";
+import { Router, Route, useRouterHistory, IndexRoute, IndexLink } from "react-router";
+import { createHashHistory } from 'history'
```
Comment (TheKevJames) [inline src/js/client.js:4]: missing semicolon

File: src/js/client.js [hunk 3]
```diff
@@ -14,11 +15,12 @@ import Profile from "./pages/Profile";
 import Info from "./pages/Info";
 
 const app = document.getElementById('app');
+const appHistory = useRouterHistory(createHashHistory)({ queryKey: false })
```
Comment (TheKevJames) [inline src/js/client.js]: missing semicolon

**Review by TheKevJames — APPROVED** [review]
See, I can review javascript too.

File: src/js/client.js (hunk 2 above)
Comment (larajanecka) [reply src/js/client.js:4]: fuck off

**Review by StevenLocke — APPROVED** [review]

**Pull request merged**